src/MBrace.Flow/CloudFlow.fs

```diff
@@ -1123,3 +1139,55 @@ module CloudFlow =
                         return! projection collector.Result
                 } }
 
+
+
+
+                
+    /// <summary>Creates a CloudFlow from the ReceivePort of a CloudChannel</summary>
+    /// <param name="channel">the ReceivePort of a CloudChannel.</param>
+    /// <returns>The result CloudFlow.</returns>
+    let ofCloudChannel (channel : IReceivePort<'T>) : CloudFlow<'T> =
```

Comment:
See comment on toCloudChannel.  Could this be named more general e.g. ``CloudFlow.ofCloudSource``?  

The contract for a IReceivePort (aka ICloudSource) is interesting - it starts N receive actions in parallel (where N is the degree-of-parallelism).